ansible.cfg

```diff
@@ -0,0 +1,3 @@
+[defaults]
+
+roles_path = provisioning/roles
```

Comment:
does this file only get applied if running ansible from top level dir or does it apply whereever you run ansible from?